Code-Hammers/code-hammers
# [CHE-7] Create EditProfile Page

### Description

This ticket is part of [CHE-2](https://codehammers.atlassian.net/browse/CHE-2?atlOrigin=eyJpIjoiNTY3YjliODk5NjY2NGJiNzg1NTdiZjk0MDYzYTc1ZjIiLCJwIjoiaiJ9) Create Profile Page story and adds an EditProfile page with basic functionality including Redux hookups and back end routing/database updates.

### Jira Task

[CHE-7](https://codehammers.atlassian.net/browse/CHE-7?atlOrigin=eyJpIjoiMWFhMWVlMjAyYzEyNGJmNWEyNWFjMTdmM2JlMDBmMWIiLCJwIjoiaiJ9)

### Testing Instructions

1. Ensure branches are up to date.
2. `git checkout CHE-7/subtask/Create-EditProfile-Page`
3. `npm run docker-dev` --> Test edit fields and ensure database updates occur.

### Checklist


### Additional Notes, Images, etc.

N/A


[CHE-2]: https://codehammers.atlassian.net/browse/CHE-2?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-7]: https://codehammers.atlassian.net/browse/CHE-7?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
